bindings/python/asv.conf.json

```diff
@@ -0,0 +1,185 @@
+{
+    // The version of the config file format.  Do not change, unless
```

Comment:
JSON doesn't support comments.

Reply:
Sorry, the comments were created by asv when initially set up. I will remove them.